reactive/jenkins.py

```diff
@@ -177,3 +198,29 @@ def migrate_charm_data():
 def stop():
     service_stop("jenkins")
     status_set("maintenance", "Jenkins stopped")
+
+
+@when("apt.installed.jenkins")
+@when("reset.jenkins.home")
+def reset_jenkins_home():
+    status_set("maintenance", "Configuring JENKINS_HOME")
+
+    home = unitdata.kv().get("storage.dir")
+    if home is None:
+        home = '/var/lib/jenkins'
+    log("Setting Jenkins home to {}".format(home))
+    unitdata.kv().set("jenkins.home", home)
+    # re-import paths to force update
+    importlib.reload(paths)
```

Comment:
I'm failing to see how the jenkins.home unitdata variable and the global variables in the paths module are related. Where is the code that connects them?